client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -57,11 +58,24 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goNext, goToStep }: Sidebar
 	const site = useSite();
 	const clipboardButtonEl = useRef< HTMLButtonElement >( null );
 	const [ clipboardCopied, setClipboardCopied ] = useState( false );
+	const { isFetchingGlobalStylesStatus, globalStylesInUse, shouldLimitGlobalStyles } =
+		usePremiumGlobalStyles( site?.ID );
```

Comment:
I think we should change this to:
```
usePremiumGlobalStyles( site?.ID ?? null );
```
Since the usePremiumGlobalStyles expects either a number or null, and `site?.ID` either resolves to a number or `undefined` which would then fail the condition on line 20 of `use-premium-global-styles.ts` :
```
	if ( siteId === null ) {
```
And would end up performing a network request with siteId=undefined.

Reply:
> `usePremiumGlobalStyles( site?.ID ?? null );`

`usePremiumGlobalStyles` defaults to `null` if the given param is undefined (or if no param is provided), so we don't need to do that.

Reply:
Technically, undefined is not compatible with `number | null` type but by adding a default null value we're hacking the type system 😛